### DISCUSSION TITLE
Browsers Crashing and Freezing the system except Safari

### DISCUSSION THREAD

**Discussion by shiodr** [ChefKissInc/NootedRed#108] [Q&A] [answered]
Is it just me or is it really a bug in AMD APUs?

**Comment by Alex-Zhangzhipeng** [comment, marked as answer]
it is a bug of nootedred, just close the hardware acceleration.

**Reply by shiodr** [reply]
ohh I see, thank you.

**Comment by ChitoKim** [comment]
I had the same issue especially in google chrome. I installed older version of chromium and it simply works well

**Reply by shiodr** [reply]
I think the only way to run the newer ones is to disable the hardware acceleration of the browser then disable graphics rasterization in chrome://flags or edge://flags
